Makefile

```diff
@@ -0,0 +1,15 @@
+LIBDIR := lib
+include $(LIBDIR)/main.mk
+
+$(LIBDIR)/main.mk:
+ifneq (,$(shell grep "path *= *$(LIBDIR)" .gitmodules 2>/dev/null))
+	git submodule sync
```

Comment:
I have a strong preference against using submodules. They’re really not a great experience. I’m willing to have to manually update this thing as needed rather than have it as a submodule.

Reply:
Ohhh. I should've read more closely. Yeah this is fine.